locales-cs-CZ.xml

```diff
@@ -251,6 +251,7 @@
     </term>
 
     <!-- VERB ROLE FORMS -->
+    <term name="container-author" form="verb"/>
```

Comment:
What is the desired behaviour with singletons? Should the processor stop fallback and return an empty string, or treat the node value as nil? 

If the latter, and this is just a placeholder, why not make it a comment instead?